examples/website/src/shared/atoms/bildwurfAdEmbed.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useEffect} from 'react'
+import {useScript} from '../utility'
+
+declare global {
+  interface Window {
+    _ASO: {
+      loadAd(param1: string, param2: string): void
+    }
+  }
+}
+
+export interface BildwurfAdEmbedProps {
+  zoneID: string
+}
+
+export function BildwurfAdEmbed({zoneID}: BildwurfAdEmbedProps) {
+  const {load} = useScript(
+    `https://media.online.bildwurf.ch/js/code.min.js`,
+    () => !!window._ASO,
+    false
+  )
+
+  useEffect(() => {
+    load()
+    try {
+      window._ASO.loadAd('bildwurf-injection-wrapper', zoneID)
+    } catch (error) {
+      console.warn('could not call _ASO.loadAd()')
```

Comment:
Do you want to specify that it's related to `bildwurf`?

Reply:
`_ASO` === Bildwurf